src/python/espressomd/lb.pxd

```diff
@@ -119,6 +119,9 @@ cdef extern from "grid_based_algorithms/lb_interpolation.hpp" namespace "Interpo
     cdef InterpolationOrder linear
     cdef InterpolationOrder quadratic
 
+cdef extern from "integrate.hpp":
+    double get_time_step()
+
```

Comment:
Is this necessary, because `integrate.pyx` is no longer a cython file? Don't we have a better place to access the `time_step`?

Reply:
Unfortunately, it is still necessary in the python branch, because the old LB implementation needs it. That's the last consumer, so this will be gone in the walberla branch.